src/demo/models/guide.js

```diff
@@ -188,25 +190,64 @@ const guides = [
     style: 'BottomMiddle',
     arrow: 'none'
   },
+  {
+    id: 'creaturesvtrash-predicting-pause4',
+    text: `Let's train A.I. again`,
+    when: {
+      appMode: AppMode.CreaturesVTrashDemo,
+      currentMode: Modes.Predicting,
+      isPaused: true,
+    },
+    style: 'BottomRight',
+    hideBackground: true
+  },
   {
     id: 'creaturesvtrash-training-init',
-    text: 'Let’s train A.I. again!',
+    text: `Now let’s teach A.I. to learn which objects should be in the water.`,
     when: {appMode: AppMode.CreaturesVTrash, currentMode: Modes.Training},
     style: 'BottomMiddle',
   },
+  {
+    id: 'creaturesvtrash-training-init2',
+    text: `In the ocean, plastic debris injures and kills fish, seabirds and marine mammals.  Keep training A.I. so we can keep the ocean clean and safe.`,
+    when: {
+      appMode: AppMode.CreaturesVTrash,
+      currentMode: Modes.Training,
+      fn: state => {
+        return state.yesCount + state.noCount >= 5;
+      }
+    },
+    style: 'RightCenter',
+    arrow: 'none',
+    image: trash6PackImage
+  },
+  {
+    id: 'creaturesvtrash-training-init3',
+    text: `Marine plastic pollution has impacted at least 267 species worldwide, including 86% of all sea turtle species and 43% of all marine mammal species. Your training of A.I. can make a difference in how well A.I. identifies trash. Keep going.`,
```

Comment:
First impression: the combination of a real fact and our instructions make it sound as though training A.I. can actually "make a difference" in the real world.

Reply:
Is that a good/bad/incorrect impression?

Reply:
Oh, it felt like an over-promise, or at least kind of a stretch.

Reply:
Fixed, per discussion.